SonarScanner.Shim/PropertiesWriter.cs

```diff
@@ -208,7 +208,13 @@ public void WriteSonarProjectInfo(string projectBaseDir, ICollection<string> sha
             AppendKeyValue(sb, SonarProperties.WorkingDirectory, Path.Combine(this.config.SonarOutputDir, ".sonar"));
             AppendKeyValue(sb, SonarProperties.ProjectBaseDir, projectBaseDir);
 
-            if (sharedFiles.Any())
+            this.projects.Select((p, index) =>
+            {
+                string moduleWorkdir = Path.Combine(this.config.SonarOutputDir, ".sonar", "mod" + index);
+                return new Tuple<ProjectInfo, string>(p, moduleWorkdir);
```

Comment:
In this case it won't change a lot but sometimes anonymous classes are easier to read than tuple. For example, here you could do the following:
`return new { Project = p, ModuleDir = moduleWorkdir };`
which you can use like `t.Project` or `t.ModuleDir` which might be easier to read than `t.Item1`.